app/course/directives/courseTable.js

```diff
@@ -456,9 +458,17 @@ let courseTable = function ($rootScope, $timeout, CourseActionCreators, $compile
             // TODO: Calculate this boolean by comparing the sum of all section seats to the plannedSeats
             var requiresAttention = false;
 
+            if (sectionGroup) {
+              requiresAttention = sectionGroup.requiresAttention;
+            }
+
             // Determine if the term is readonly
             var cellClass = sectionGroupId ? "sg-cell is-offered" : "sg-cell";
 
+            if (state.uiState.selectedCourseId == courseId && state.uiState.selectedTermCode == termCode) {
+              cellClass += " selected-td";
```

Comment:
A space? A space?!

Surely the classes should be stored as a list, and joined together via a space!

Think of all the future expansions to cellClass, all the things it could be!

Oh the humanity!

Reply:
👋

Reply:
Oooh, a wild Chris appears!

Also I agree with Chris 👍

Reply:
👋

I blame the person that wrote line 421.